src/rust/bitbox-aes/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2025 Shift Cryptos AG
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[package]
+name = "bitbox-aes"
+version = "0.1.0"
+authors = ["Shift Crypto AG <support@bitbox.swiss>"]
+edition = "2024"
+license = "Apache-2.0"
+
+[dependencies]
+bitbox02 = { path = "../bitbox02", optional = true }
+sha2 = { workspace = true }
+hmac = { version = "0.12", default-features = false, features = ["reset"] }
+ctaes = { git = "https://github.com/BitBoxSwiss/rust-ctaes", rev = "0df5295d9afe2da49e8bbe5eba8fd74e2d53f8ed" }
```

Comment:
Once https://github.com/BitBoxSwiss/rust-ctaes/pull/1 is merged, I could make a tag and reference that, or even push it to crates.io and get it from there instead.

Reply:
yeah, push to crates.io sounds good to me.

Reply:
Done and updated this PR.